analyzers/src/SonarAnalyzer.Common/Rules/SymbolicExecutionRunnerBase.cs

```diff
@@ -133,7 +133,10 @@ public SymbolicRuleCheck CreateInstance(IAnalyzerConfiguration configuration, So
             }
 
             public object CreateSonarFallback(IAnalyzerConfiguration configuration) =>
-                configuration.ForceSonarCfg && createSonarFallbackInstance is not null ? createSonarFallbackInstance() : null;
+                createSonarFallbackInstance is not null
+                && (configuration.ForceSonarCfg || !ControlFlowGraph.IsAvailable)
```

Comment:
`IsAvailable` is not testable, as we always run on the latest roslyn. It's tested via presence of results in ITs